### PR TITLE
Hello Rails Back End 

### PR DESCRIPTION
**Pull request by Bantealem**
- [x] In this pull request i have done the following 
- [x] Created a new Rails API app called 'hello-rails-back-end'.
- [x] Initialized my project with Git.
- [x] Make sure that my project has a Postgres database set up.
- [x] Created a table for storing my messages and create 5 different greetings.
- [x] Create an API endpoint that selects a random greeting from my table (you will need a controller with an action and rails route).

### CODE REVIEW

**Review by rotimiazeez — COMMENTED** [review]
# Hi @Bantealem , 

### While you made a great effort in this project, I cannot proceed to review your code.

## Invalid Code Review Request

### Please make sure the frontend part of your project has a professional and descriptive README, and in the README file for the fronted, there should find a link to the backend part 👍🏽 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something needs to be 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

**Review by Ugboaja-Uchechi — COMMENTED** [review]
Hi @Bantealem , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### **Like the previous reviewer mentioned kindly add a link to the Frontend of your project to your README file** :pray:

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by Bantealem** [comment]
Hi @Ugboaja-Uchechi 
I have added in the feature branch and I didn't merge the pull request.
Thank you.

File: config/routes.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+Rails.application.routes.draw do
+  # get 'root/index'
+  # root 'root#index'
+  namespace :api do
+    namespace :v1 do
+      get 'greetings', to: 'greetings#index'
+      # post 'greetings', to: 'greetings#create'
+      # get 'greetings/:id', to: 'greetings#show'
+      # patch 'greetings/:id', to: 'greetings#update'
+      # delete 'greetings/:id', to: 'greetings#destroy'
```
Comment (FrederickMih) [inline config/routes.rb:10]: - Commented code renders our codebase messy. Do not push commented code to the repo. It's ugly. Kindly remove all comments in the routes file.

**Review by FrederickMih — APPROVED** [review]
## Hi :raising_hand: @Bantealem , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights :green_circle: 

- Two apps work together :heavy_check_mark: 
- PR is descriptive :heavy_check_mark: 
- All Linter checks passed :heavy_check_mark: 

## Optional suggestions

**N/A**

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**